src/main/java/be/seeseemelk/mockbukkit/inventory/meta/ItemMetaMock.java

```diff
@@ -19,7 +21,7 @@
 import static java.util.Objects.nonNull;
 
 @SuppressWarnings("deprecation")
-public class ItemMetaMock implements ItemMeta, Damageable {
+public class ItemMetaMock implements ItemMeta, Damageable, ConfigurationSerializable {
```

Comment:
ItemMeta already extends `ConfigurationSerializable` so this would be somewhat redundant. Is there any particular reason you implemented it here too?

Reply:
Ah my bad! Removing it and replacing the ArrayLists in the next commit.